druid/src/widget/split.rs

```diff
@@ -316,3 +355,15 @@ impl<T: Data> Widget<T> for Split<T> {
         self.child2.paint_with_offset(paint_ctx, &data, env);
     }
 }
+
+// Move to std lib clamp as soon as https://github.com/rust-lang/rust/issues/44095 lands
+fn clamp(mut x: f64, min: f64, max: f64) -> f64 {
+    assert!(min <= max);
+    if x < min {
```

Comment:
Not a blocker, but I would probably just write this as `x.max(min).min(max)`?

Reply:
I just did it how Rust (nightly) did it, to keep compatibility in edge-cases. There is quite some [discussion](https://github.com/rust-lang/rfcs/pull/1961#issuecomment-289893089) going on what is the proper way to write clamp. Seems like at edge cases weird things happens with these small (seemingly) similar implementations.